tests/integration/botocore/test_sts.py

```diff
@@ -40,29 +40,3 @@ def test_regionalized_endpoints(self):
         # Signing error will be thrown with the incorrect region name included.
         with self.assertRaisesRegex(ClientError, 'ap-southeast-1') as e:
             sts.get_session_token()
-
-
-def test_assume_role_with_saml_no_region_custom_endpoint(patched_session):
-    # When an endpoint_url and no region are given, AssumeRoleWithSAML should
-    # resolve to the endpoint_url and succeed, not fail in endpoint resolution:
-    # https://github.com/aws/aws-cli/issues/7455
-
-    client = patched_session.create_client(
-        'sts', region_name=None, endpoint_url="https://custom.endpoint.aws"
-    )
-    assert client.meta.region_name is None
-
-    mock_response_body = b"""\
-<AssumeRoleWithSAMLResponse xmlns="https://sts.amazonaws.com/doc/2011-06-15/">
-    <AssumeRoleWithSAMLResult></AssumeRoleWithSAMLResult>
-</AssumeRoleWithSAMLResponse>
-"""
-    with ClientHTTPStubber(client) as http_stubber:
```

Comment:
We should remove the added import of `ClientHTTPStubber`. It got added as part of the original port: https://github.com/aws/aws-cli/pull/7465

Reply:
Removed it, thanks!